kmath-tensors/src/commonMain/kotlin/space/kscience/kmath/tensors/api/TensorAlgebra.kt

```diff
@@ -60,15 +61,14 @@ public interface TensorAlgebra<T>: Algebra<Tensor<T>> {
      *
      * @param value the number to be added to each element of this tensor.
      */
-    public operator fun Tensor<T>.plusAssign(value: T): Unit
+    public operator fun Tensor<T>.plusAssign(value: T)
```

Comment:
aren't return types required for library mode?

Reply:
No, you don't have to specify `Unit` in library mode.

Reply:
OK then.